metabase/metabase
# [MLv2] [FE] Filter modal segments

Epic https://github.com/metabase/metabase/issues/34112

Changes:
- Adds segments to the filter modal. They are available in the first group (tab).
- Make sure they can be filtered by using the top-level search in the modal. 
- Converts multi-value pickers to `MultiSelect` from `mantine`. It's not styled yet, this is TBD separately. Also handling field values TBD in a separate PR.

How to verify:
- You can add a segment as a filter and remove it from the query. Query should be modified and work correctly.

<img width="1143" alt="Screenshot 2023-12-12 at 16 23 00" src="https://github.com/metabase/metabase/assets/8542534/961259b5-d543-465b-9631-f9f91d5d78ec">
